person.rb

```diff
@@ -1,12 +1,24 @@
-class Person
+class Nameable
+  def correct_name(name)
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Kindly implement this class in a separate file so you can import it when you use it, instead of writing the same class twice in two different files.